src/Symfony/Bundle/TwigBundle/Controller/PreviewErrorController.php

```diff
@@ -47,7 +47,6 @@ public function previewErrorPageAction(Request $request, $code)
             '_controller' => $this->controller,
             'exception' => $exception,
             'logger' => null,
-            'format' => $request->getRequestFormat(),
```

Comment:
The correct attribute is called `_format`, but it is not needed anyway: the `duplicate()` method already does it:
https://github.com/symfony/symfony/blob/b2f5b8a1e2e704e9c31ce08d1a28ca116ae8e7fb/src/Symfony/Component/HttpFoundation/Request.php#L460-L466

Reply:
Fixed in 3.4 https://github.com/symfony/symfony/pull/32694